package.json

```diff
@@ -33,13 +34,15 @@
     "graphql-tag": "1.2.4",
     "isomorphic-fetch": "2.2.1",
     "lodash.merge": "4.6.0",
+    "lodash.omit": "4.5.0",
+    "lodash.pick": "4.4.0",
     "lodash.pickby": "4.6.0",
     "pluralize": "3.1.0",
     "react-router": "3.0.2",
     "react-router-redux": "4.0.8",
     "redux-saga": "0.14.3"
   },
   "peerDependencies": {
-    "admin-on-rest": "~0.8.0"
+    "admin-on-rest": "^0.8.1"
```

Comment:
It serves different purpose:

`peerDependencies` indicates to the end user that this package is depending on aor 
However, npm and yarn do not install peer deps in development mode

`devDependencies` indicates to the developer contributing to this package that it needs aor to run tests for example